docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -144,27 +179,32 @@ Add the following new bundle to your new kernel file, `<new-ez-root>/app/AppKern
+Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform backend UI, allowing you to take full advantage of what the Platform has to offer.
 
-##### 3.2. Execute XmlText Migration script
+###### 3.2.1 Migrate XmlText content to RichText
 
-For migrating content from the XmlText format to the new RichText format, a migration script exists, execute the following from &lt;new-ez-root&gt;:
+**If** you are ready to migrate your eZ Publish XMLText content to the eZ Platform RichText format and start using pure eZ Platform setup, you can use a script to migrate content from the XmlText format to the new RichText format. Execute the following from &lt;new-ez-root&gt;:
 
 `php app/console ezxmltext:convert-to-richtext -v`
 
-The migration script is currently in beta, which is why command example is suggested with verbose flag. Feedback on how it works and on how we can improve it is welcome [on the repository](https://github.com/ezsystems/ezplatform-xmltext-fieldtype).
+The command example suggests using the verbose flag. This is optional, but recommended as we are actively gathering feedback on how it works with older eZ Publish content, and on how we can improve it both [via issues in the repository](https://github.com/ezsystems/ezplatform-xmltext-fieldtype), and on Slack.
+
+###### 3.2.2 Migrate Page field to Landing Page (eZ Enterprise only)
+
+**If** you use Page field (ezflow) and an eZ Enterprise subscription, and are ready to migrate your eZ Publish Flow content to the eZ Enterprise Landing Page format. You can use a script to migrate your Page content to Landing Page, to start using a pure eZ Enterprise setup. See [Migrating legacy Page field (ezflow) to Landing Page (Enterprise)](#migrating-legacy-page-field-ezflow-to-landing-page-enterprise) for more information.
 
-##### 3.3. Migrate Page field to Landing Page (eZ Enterprise only)
+###### 3.2.3 Add Date Based Publisher schema (eZ Enterprise only)
 
-If you have Page field (ezflow) content and an eZ Enterprise subscription, you can use a script to migrate your Page content to eZ Enterprise Landing Page. See [Migrating legacy Page field (ezflow) to Landing Page (Enterprise)](#migrating-legacy-page-field-ezflow-to-landing-page-enterprise) for more information.
+For date based publisher feature, there is an additional table, you can import it to your database using the following sql file:
+`<new-ez-root>/vendor/ezsystems/date-based-publisher/bundle/Resources/install/datebasedpublisher_scheduled_version.sql`
```

Comment:
I'll need dev & doc team to own this section, what else needs to be added here for migration to EE install with Studo?

Customer that uncovered this part was missing now also is reporting missing table from: `vendor/ezsystems/ezstudio-form-builder/bundle/Resources/install/form_builder.sql`. What else need to be mentioned here? Is there one single place in installer we can link to / always be able to verify against to make sure we have all? /cc @lserwatka